webpack/components/extensions/HostDetails/Tabs/TracesTab.js

```diff
@@ -43,20 +85,29 @@ const TracesTab = () => {
         fetchItems={fetchItems}
         autocompleteEndpoint={`/hosts/${hostId}/traces/auto_complete_search`}
         foremanApiAutoComplete
+        displaySelectAllCheckbox
+        selectPage={selectPage}
+        selectNone={selectNone}
+        selectAll={selectAll}
         rowsCount={results?.length}
+        areAllRowsOnPageSelected={areAllRowsOnPageSelected}
+        areAllRowsSelected={areAllRowsSelected}
         variant={TableVariant.compact}
         status={status}
         metadata={meta}
+        selectedCount={selectedTraces.size}
+        modelName="trace"
       >
         <Thead>
           <Tr>
+            <Th />
```

Comment:
Empy row, but why?

Reply:
It's the checkbox column header!

Reply:
I can't understand in the code how the select all is working..Would you need any updates to the rails traces controller or is it already supported?

Reply:
No updates to the backend were needed.  We just keep track on the front end of which trace IDs are selected, and render accordingly.

Reply:
Backend updates will be needed to get 'Select all' functionality working, however. That is left for a future PR.

Reply:
> I can't understand in the code how the select all is working

To be clear, the 'Select all' checkbox is correctly rendering as checked, but the 'Select all' menu item doesn't work yet.

The checkbox rendering works just by comparing the number of selected items and the total from pagination metadata.